### PR TITLE
Don't delete images or iso which are used in a VM.

### PR DESCRIPTION
**Pull request by renlinch**

### CODE REVIEW

File: root/src/storage.html [hunk 1]
```diff
@@ -106,9 +106,15 @@ <h3>{% i18n "Images" %}</h3>
                                         <td style="width:30px;">
                                             <form action="" method="post" role="form">{{ csrf_token }}
                                                 <input type="hidden" name="volname" value="{{ volume.name }}">
+                                                {% ifnotequal volume.usedby "" %}
```
Comment (dantti) [inline root/src/storage.html]: Please avoid ifequal or ifnotequal, if volume.usedby != "" is easier to read

File: src/lib/storagevol.cpp [hunk 2]
```diff
@@ -55,6 +55,45 @@ QString StorageVol::size()
...
+{
+    QString usedbyvm;
+    QString pathdisk = path();
+    QString tryname;
+    QString trysrc;
+    QVector<Domain *> ret;
+    
+    m_conn = virStorageVolGetConnect(m_vol);
+    virDomainPtr *domains;
+    int count = virConnectListAllDomains(m_conn, &domains, VIR_CONNECT_LIST_DOMAINS_ACTIVE | VIR_CONNECT_LIST_DOMAINS_INACTIVE);
```
Comment (dantti) [inline src/lib/storagevol.cpp:68]: Can't you go up on the parent() and get the Connection class which will give you Domains objects and there you get this, would be less work or less duplicated work.

Comment (renlinch) [reply src/lib/storagevol.cpp:68]: that could be my to less knowledge in c++ .. i was trying to get that with the available routines but didn't get it to work... and at the end i was thinking this is doing less work than your complex routines and was doing it this way. The work was inspired by the virt-manager...

Comment (dantti) [reply src/lib/storagevol.cpp:68]: ok, I can refactor this later anyway

**Review by renlinch — COMMENTED** [review]
changed

**Comment by renlinch** [comment]
should i send a request for some other changes ifnotequal to if ? 
like this one : 
` {% ifnotequal volume.type "iso" %}`
i was copy my ifnotequal from there ...

**Comment by dantti** [comment]
Sure, if you can, the HTML came from webvirtmgr which used ifequal (which afaik is even deprecated in Django), I changed what I saw, but there was more...
Want me to leave this open for other ifequal changes?

**Comment by renlinch** [comment]
i prefer a new request... would be cleaner...

**Pull request merged**